libs/components/forms/src/lib/modules/input-box/input-box.component.spec.ts

```diff
@@ -778,6 +778,27 @@ describe('Input box component', () => {
       expect(els.inputEl?.hasAttribute('aria-describedby')).toBeFalse();
     });
 
+    it('should allow a child to add hint text programmatically', () => {
+      const fixture = TestBed.createComponent(InputBoxFixtureComponent);
+      fixture.detectChanges();
+
+      const els = getDefaultEls(fixture, 'input-host-service');
+
+      expect(els.hintTextEl).toHaveText('Host component hint text.');
+    });
+
+    it('should allow both child and consumer specified hint text.', () => {
```

Comment:
Probably don't need the closing period `.`.

Reply:
Fixed